pom.xml

```diff
@@ -18,6 +18,8 @@
         <java.version>11</java.version>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <driver.jdbc.group.id>mysql</driver.jdbc.group.id>
```

Comment:
@mcruzdev this is not actually solving the problem, what I would do for now is to leave it with mysql until we figure out how to add the driver more on the docker side.